contracts/AxelarGateway.sol

```diff
@@ -50,13 +50,21 @@ contract AxelarGateway is IAxelarGateway, AdminMultisigBase {
     // solhint-disable-next-line var-name-mixedcase
     address internal immutable AUTH_MODULE;
     // solhint-disable-next-line var-name-mixedcase
+    address internal immutable GOVERNANCE_MODULE;
+    // solhint-disable-next-line var-name-mixedcase
     address internal immutable TOKEN_DEPLOYER_IMPLEMENTATION;
 
-    constructor(address authModule_, address tokenDeployerImplementation_) {
+    constructor(
+        address authModule_,
+        address governanceModule_,
+        address tokenDeployerImplementation_
+    ) {
         if (authModule_.code.length == 0) revert InvalidAuthModule();
+        if (governanceModule_.code.length == 0) revert InvalidGovernanceModule();
```

Comment:
Don't think we need to enforce that it's a contract. Could also call it `governance`/`governor` so it feels more generic instead of a module. Also simplifies testing a bit.

Reply:
Agree. In that sense, just `GOVERNANCE` instead of `GOVERNANCE_MODULE`.